multicluster/hack/update-codegen-dockerized.sh

```diff
@@ -21,7 +21,44 @@ set -o pipefail
 GOPATH=`go env GOPATH`
 ANTREA_PKG="antrea.io/antrea"
 
-cd multicluster
+function generate_mocks {
+  # Generate mocks for testing with mockgen.
+  MOCKGEN_TARGETS=(
+    "multicluster/controllers/multicluster MemberClusterStatusManager,RemoteCommonAreaGetter ."
+    "multicluster/controllers/multicluster/commonarea CommonArea,RemoteCommonArea . mock_remote_common_area.go"
+  )
+
+  # Command mockgen does not automatically replace variable YEAR with current year
```

Comment:
Could we share the code with `hack/update-codegen-dockerized.sh` in antrea root?

Reply:
I think it should work as well, I will double check.

Reply:
Done